onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -990,7 +990,43 @@ ::std::vector<::std::basic_string<ORTCHAR_T>> GetParameterStrings() {
   return v;
 }
 
+auto GenerateCustomTestName  = [](const ::testing::TestParamInfo<ModelTest::ParamType>& info) {
```

Comment:
thinking there could be a better name? 
these are not really custom tests. 
something like ExpandModelName or GenerateModelName ?

Reply:
Changed it to ExpandModelName